src/utils.js

```diff
@@ -0,0 +1,4 @@
+// @flow
+
+export const PLACEHOLDER_REGEX = /xxx(\S)xxx/gm
```

Comment:
Yeah just define it in place each time. It's simple enough. Alternately, export a function that returns the regex

The rest looks great!